jdaviz/configs/default/plugins/export/export.py

```diff
@@ -207,6 +220,17 @@ def _set_subset_not_supported_msg(self, msg=None):
         else:  # no subset selected (can be '' instead of None if previous selection made)
             self.subset_invalid_msg = ''
 
+    def _set_dataset_not_supported_msg(self, msg=None):
+        if self.dataset.selected_obj is not None:
+            if self.dataset.selected_obj.meta.get("Plugin", None) is None:
+                self.data_invalid_msg = "Data export is only available for plugin generated data."
+            elif not isinstance(self.dataset.selected_obj, (Spectrum1D, CCDData)):

```

Comment:
If the data doesn't come from a plugin, it can't be exported (yet). If it does come from a plugin, it must either be a Spectrum1D or CCDData. Is this the output format of all plugin data currently, would this ever be encountered? (cc @kecnry).